style.css

```diff
@@ -601,11 +600,13 @@ input:focus:invalid {
  #container1 {
     height: 100vh;
     background-color: #1c1a19;
-    background-image: url("images/Illustration-header.png");
+    background-image: url(images/Illustration-header.png);
     background-repeat: no-repeat;
     background-position: right;
+    background-position-y: 0.2rem;
     background-size: 40%;
```

Comment:
- [ ] As requested by previous reviewer I would suggest you please stick to the design as much as possible. Please adjust the spacing by looking at the design below.
### Your design
![his](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/106e6dba-a3a6-47c5-a777-b1b4cf35f7dc)

### Figma design
![fimga](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/58cd8f87-9394-4574-84c5-78eceefea646)